sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -91,7 +91,7 @@ cognitive services.
     from azure.ai.textanalytics import TextAnalyticsClient, TextAnalyticsApiKeyCredential
     
     credential = TextAnalyticsApiKeyCredential("<api_key>")
-    text = TextAnalyticsClient(endpoint="https://westus2.api.cognitive.microsoft.com/", credential=credential)
+    text = TextAnalyticsClient(endpoint="https://<region>.api.cognitive.microsoft.com/", credential=credential)
```

Comment:
I think that somewhere in this section we should add a note about how people can find the endpoint.
Maybe saying to go to the Azure Portal and look for the endpoint of the resource, or use the Azure CLI....

Reply:
The section right below this explains how to look up the endpoint :)